airflow/config_templates/config.yml

```diff
@@ -944,6 +944,14 @@ metrics:
   description: |
     StatsD (https://github.com/etsy/statsd) integration settings.
   options:
+    metrics_use_pattern_match:
+      description: |
+        If true, metrics_allow_list and metrics_block_list will use regex pattern matching
```

Comment:
I would suggest changing description of "metrics_allow_list" and/or "metrics_block_list" to mention about behaviour change when this is "True"

Reply:
I'll workshop a couple of ideas.  Thanks for the suggestion

Reply:
Revised